manticore/core/manticore.py

```diff
@@ -356,7 +370,60 @@ def __init__(self, initial_state, workspace_url=None, outputspace_url=None, **kw
 
         # Workers will use manticore __dict__ So lets spawn them last
         self._workers = [self._worker_type(id=i, manticore=self) for i in range(consts.procs)]
-        self._is_main = True
+        self._snapshot = None
+        self._main_id = os.getpid(), threading.current_thread().ident
+
+    def is_main(self):
+        ''' True if called from the main process/script
```

Comment:
Pretty sure `black` should be flagging these. Looking at the Lint results it seems that something might be broken. I'll see if I can fix it in another PR.

Reply:
blkn